index.bs

```diff
@@ -1053,8 +1058,7 @@ that determine how WebTransport connection is established and used.
 To <dfn>compute a certificate hash</dfn>, do the following:
```

Comment:
This should really take the algorithm as an input.  For another issue, perhaps.

Reply:
Filed #359.